### DISCUSSION TITLE
Got error 404: "The specified key does not exist."

### DISCUSSION THREAD

**Discussion by abdallahheidar** [cloudbet/docs#73] [Q&A]
Hi All,

Tried using the existing API Key located at https://www.cloudbet.com/en/player/api and does not work.

When copied the curl test command to the cmd on windows, got error:
`invalid api key
curl: (6) Could not resolve host: application
curl: (3) URL using bad/illegal format or missing URL`

Also tried the following python code to test API and get account funds, and got the error: 

> <?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: kubershmuber-prod-eu-api-docs/api/api/v1/account/balance</Details></Error>

Code:
```
import requests

# Cloudbet API endpoints
ACCOUNT_API_URL = "https://api.cloudbet.com/api/v1/account"

# Get account balance
def get_account_balance():
  headers = {
    "Authorization": "Bearer " + API_KEY
  }
  response = requests.get(ACCOUNT_API_URL + "/balance", headers=headers)
  if response.status_code == 200:
    return response.json()["balance"]
  else:
    return response.status_code, response.text

account_balance = get_account_balance()
print(account_balance)

```

I would appreciate your assistance.

**Comment by kgravenreuth** [comment]
Hi @abdallahheidar ,

Thanks for getting in touch! 

There are a couple of issues with that script:

- The `ACCOUNT_API_URL` should be `https://sports-api.cloudbet.com/pub/v1/account` as per our [Account API Docs](https://www.cloudbet.com/api/?urls.primaryName=Account#/PlayerAccount/accountBalance)
- The path should be `/currencies/{currency}/balance`, e.g. `/currencies/BTC/balance` as per these docs.
- The API Key should be sent as the value of the `X-API-Key` header, i.e. `X-API-Key: {API_KEY}` rather than a Bearer token.

Please try again after making these adjustments and let us know if you have any other questions. 

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
